ustriage/task.py

```diff
@@ -161,9 +162,9 @@ def compose_pretty(self, shortlinks=True, extended=False):
             )
             bug_url = format_string % self.url
 
-        flags = self.get_flags()
+        flags = self.get_flags(newbug)
 
-        text = '%s - %3s %-13s %-19s' % (
+        text = '%s - %4s %-13s %-19s' % (
             bug_url,
             flags,
```

Comment:
The flags variable isn't used any other places, so maybe consider just inlining the get_flags() call here and eliminating the variable?

Reply:
Agreed, IIRC this come from an older control flow in the code.
Fixed - thanks for the suggestion.